### DISCUSSION TITLE
How do you get Disscussions?

### DISCUSSION THREAD

**Discussion by FaGru3n** [eclipse-tractusx/tractusx-edc#256] [Q&A] [answered]
Hi 👋🏼 ,

personal i find discussions cool and just want to know you get it?

my assumtion is to raise a Eclipse Ticket?

**Comment by stephanbcbauer** [comment]
@FaGru3n EF Ticket for what kind of topic? Sorry, i don't get your point

**Reply by FaGru3n** [reply]
Hi @stephanbcbauer, 
i have seen that this repository is the only one with the discussion section so far, so just want to know how and where do you get the discussion section. 😉 

My Goal --> I want to enable it for another repo within etx GitHub Organsitation

**Reply by stephanbcbauer** [reply]
@FaGru3n ah, sorry. Understood. To be honest, I don't remember. But I think we requested it at the system team.

**Reply by FaGru3n** [reply]
i´m from system team as well but maybe missed something 👍🏼

**Reply by FaGru3n** [reply]
will raise an issue to Eclipse as GitHub Organisation Owner and will share the information.

**Reply by stephanbcbauer** [reply]
> will raise an issue to Eclipse as GitHub Organisation Owner and will share the information.

Maybe you can tell us, what your solution was ;) thx

**Reply by FaGru3n** [reply, marked as answer]
My suggestion was right, to get disscussions on your product repository within Eclipse Tractus-X open an GitLab Issue against Eclipse Foundation and they can change your repository settings. 

Reference GitLab Issue:
https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/3007#note_1110203
